style.css

```diff
@@ -1,3 +1,3 @@
 h1 {
-    color: cyan;
-}
\ No newline at end of file
+  color: cyan;
```

Comment:
### Optional
- [ ] It would be better if we capitalize our commit messages going forward, for example; write `Fix linter` instead of `fix linter`, etc